### ISSUE TITLE
crash in canRemoveVertex

### ISSUE THREAD

**Issue by wenxiu2199** [recastnavigation/recastnavigation#508]
see DetourTileCacheBuilder.cpp
in canRemoveVertex

// Find edges which share the removed vertex.
unsigned short edges[MAX_REM_EDGES];

I compare the func in RecastMesh.cpp
it seem to be  edges[MAX_REM_EDGES * 3]

**Comment by SpangMcgraw** [comment]
Yup have hit this also, it seems edges should be declared as edges[MAX_REM_EDGES * 3]

**Comment by grahamboree** [comment]
@SpangMcgraw Do you have a repro case I can use to test this?
